README.md

```diff
@@ -23,11 +23,16 @@ We're careful about publishing [information collected during research](https://h
 
 ## Development
 
-``` bash
-git clone https://github.com/18F/handbook.git
```

Comment:
This was suggesting using HTTPS for cloning, but we probably want to encourage SSH over HTTPS.